### PR TITLE
Miscellaneous New and Noteworthy fixes for 4.27

### PR DESCRIPTION
**Pull request by alshamams**
-fix incorrect versioning
-load missing images (End user reported)
-fix incorrect html tagging

/cc @lshanmug

### CODE REVIEW

File: bundles/org.eclipse.jdt.doc.user/whatsNew/jdt_whatsnew.html [hunk 1]
```diff
@@ -27,18 +27,18 @@ <h2>What's New in Eclipse 4.27 (JDT)</h2>
 They are grouped into:</p>
 <ul> <!-- NOTE: Sync ../topics_WhatsNew.xml with this! -->
   	<!--li><a href="#Java19">Java 19 Support</a></li-->
-	<li><a href="#JavaCompiler">Java Compiler</a></li>
-  	<li><a href="#JUnit">JUnit</a></li>
-	<li><a href="#JavaEditor">Java Editor</a></li>
+	<li><a href="#JavaCompiler" target="_blank">Java Compiler</a></li>
```
Comment (lshanmug) [inline bundles/org.eclipse.jdt.doc.user/whatsNew/jdt_whatsnew.html:30]: target="_blank" is added when the link can't open in the same frame and has to be opened in a new tab. It's not required in these places. Any reason it's added?

File: bundles/org.eclipse.jdt.doc.user/whatsNew/jdt_whatsnew.html [hunk 2]
```diff
@@ -172,7 +172,7 @@ <h2>Java Editor </h2>
 
      <p>Support has been added in Eclipse to recognize the inline version of the tag and show the proper Javadoc hover.</p>
 
-     <p>For details on the inline <b>@return</b> tag, see: <a href="https://docs.oracle.com/en/java/javase/19/docs/specs/javadoc/doc-comment-spec.html#return" target="_blank">Javadoc Comment Spec</a></p>
```
Comment (lshanmug) [inline bundles/org.eclipse.jdt.doc.user/whatsNew/jdt_whatsnew.html:175]: target="_blank" is required, should not be removed.

File: bundles/org.eclipse.jdt.doc.user/whatsNew/jdt_whatsnew.html [hunk 3]
```diff
@@ -201,7 +201,7 @@ <h2>JDT Developers</h2>
       All four views are actually not new, however they were previously only available for installation 
       via Eclipse Marketplace and not included in the SDK itself.
       Now they are shipped with SDK package and also will be available for installation in other packages
-      via <a href="https://download.eclipse.org/eclipse/updates/latest/" target="_blank">https://download.eclipse.org/eclipse/updates/latest/</a>
```
Comment (lshanmug) [inline bundles/org.eclipse.jdt.doc.user/whatsNew/jdt_whatsnew.html:204]: target="_blank" is required, should not be removed.

File: bundles/org.eclipse.pde.doc.user/whatsNew/pde_whatsnew.html [hunk 4]
```diff
@@ -17,21 +17,21 @@
...
+for the 4.27 release of Eclipse. They are grouped into:</p>
 <ul> <!-- NOTE: Sync ../topics_WhatsNew.xml with this! -->
 	<!--li><a href="#dialogs-wizards-views">Dialogs, Wizards and Views</a></li-->
 	<!--li><a href="#editors">Editors</a></li-->
 	<!--li><a href="#APITools">API Tools</a></li-->
 	<!--li><a href="#pde-launching">PDE Launching</a></li-->
 	<!--li><a href="#pde-compiler">PDE Compiler</a></li-->
 	<!--li><a href="#api">API Enhancements</a></li-->
-	<li><a href="#GeneralUpdates">General Updates</a></li>
+	<li><a href="#GeneralUpdates" target="_blank">General Updates</a></li>
```
Comment (lshanmug) [inline bundles/org.eclipse.pde.doc.user/whatsNew/pde_whatsnew.html:34]: target="_blank" not required

File: bundles/org.eclipse.platform.doc.isv/whatsNew/platform_isv_whatsnew.html [hunk 5]
```diff
@@ -26,14 +26,14 @@ <h2>What's New in Eclipse 4.27</h2>
 made to the Eclipse Platform and SWT for the 4.27 release of Eclipse.
 </p>
 <ul><!-- NOTE: Sync ../topics_WhatsNew.xml with this! -->
-  <li><a href="#Platform">Platform Changes</a></li>
-  <!--li><a href="#SWT">SWT Changes</a></li-->
+  <li><a href="#Platform" target="_blank">Platform Changes</a></li>
```
Comment (lshanmug) [inline bundles/org.eclipse.platform.doc.isv/whatsNew/platform_isv_whatsnew.html:29]: target="_blank" not required

File: bundles/org.eclipse.platform.doc.isv/whatsNew/platform_isv_whatsnew.html [hunk 6]
```diff
@@ -26,14 +26,14 @@ <h2>What's New in Eclipse 4.27</h2>
...
+  <!--li><a href="#SWT" target="_blank">SWT Changes</a></li-->
 </ul>
 
 <p>
 New features oriented towards end-users of the platform
 can be viewed in the
-<a href="../../org.eclipse.platform.doc.user/whatsNew/platform_whatsnew.html">What's New</a>
+<a href="../../org.eclipse.platform.doc.user/whatsNew/platform_whatsnew.html" target="_blank">What's New</a>
```
Comment (lshanmug) [inline bundles/org.eclipse.platform.doc.isv/whatsNew/platform_isv_whatsnew.html:36]: target="_blank" not required

**Review by lshanmug — CHANGES_REQUESTED** [review]

**Comment by alshamams** [comment]
@lshanmug 
I have made the required changes. Kindly review and let me know.

**Comment by alshamams** [comment]
@lshanmug 
Kindly look through and let me know if there are any other edits.

**Review by lshanmug — APPROVED** [review]

File: bundles/org.eclipse.pde.doc.user/topics_WhatsNew.xml [hunk 7]
```diff
@@ -6,7 +6,6 @@
      are made available under the terms of the Eclipse Public License v1.0
      which accompanies this distribution, and is available at
      http://www.eclipse.org/legal/epl-v10.html
-
```
Comment (lshanmug) [inline bundles/org.eclipse.pde.doc.user/topics_WhatsNew.xml:9]: Please don't change unrelated parts in the PR. Please review the diff in Eclipse before updating the PR.

File: bundles/org.eclipse.pde.doc.user/whatsNew/pde_whatsnew.html [hunk 8]
```diff
@@ -70,4 +70,4 @@ <h2>What's New in 4.26 (PDE)</h2>
 <!-- ****************** END OF N&N TABLE ****************** -->
 
 </body>
-</html>
```
Comment (lshanmug) [inline bundles/org.eclipse.pde.doc.user/whatsNew/pde_whatsnew.html:73]: Please don't change unrelated parts in the PR

**Review by lshanmug — CHANGES_REQUESTED** [review]

**Review by lshanmug — APPROVED** [review]

**Comment by MohananRahul** [comment]
Tracked #995

**Pull request merged**